### ISSUE TITLE
the supported version is too old

### ISSUE THREAD

**Issue by zhubingbing** [tcpcloud/horizon-rally-dashboard#3]
The supported version is too old
1. horizon conf is too old ,for exemple: settings.COMPRESS_OFFLINE_CONTEXT['HORIZON_CONFIG'] = conf.HORIZON_CONFIG
COMPRESS_OFFLINE_CONTEXT should is HORIZON_COMPRESS_OFFLINE_CONTEXT_BASE
2. not supprt rally 0.9

**Comment by michaelkuty** [comment]
Yes, it is, this package need some update around rally integration
